service/src/test/java/uk/gov/hmcts/reform/fpl/service/ApplicantServiceTest.java

```diff
@@ -130,4 +137,45 @@ void shouldNotAddNewPartyIdWhenApplicantsAlreadyHasPartyIdValue() {
 
         assertThat(applicant.getParty().partyId).isEqualTo(uuid);
     }
+
+    @Test
+    void shouldReturnApplicantCollectionWithOrganisationDetailsWhenOrganisationExists() {
+        CaseData caseData = CaseData.builder().build();
+
+        List<Applicant> applicants = unwrapElements(service.expandApplicantCollection(
+            caseData, buildOrganisation()));
+
+        assertThat(applicants.get(0).getParty().getOrganisationName())
+            .isEqualTo(buildOrganisation().getName());
+        assertThat(applicants.get(0).getParty().getAddress()).isEqualTo(buildOrganisation()
+            .getContactInformation().get(0).toAddress());
+    }
```

Comment:
I would extract buildOrganisation() to a variable in this test rather than call it 3 times

Reply:
makes sense, done :)